Results_LW8.R

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env Rscript
+# Author: OLinares-Perdomo
+# Usage: Results.txt file for LW8 (The ParFastas script and Cecret-wf must be executed before)
+# Parameter: Run Name ;  i.e. Combining_Results.R UT-VH0770-250110
+# last modified on 2025-01-15
+
+library("data.table");library("dplyr");library("utils");
+library("lubridate");find("read.table");library("readr"); 
```

Comment:
Are these installed in prod already?

Reply:
The read.table and utils packages are part of the base R, while the non-base packages lubridate, ready and dplyr have been successfully installed in the production environment.

Reply:
readr,  sorry for the typo in the previous response.